Kin Unity/Assets/Kin/Editor/PostProcessor.cs

```diff
@@ -1,4 +1,5 @@
-﻿using UnityEngine;
+﻿#if UNITY_IOS
```

Comment:
Why did you need to add this?
I have a task to do this as well, DP-582

Reply:
cause Unity trying to build it even if build target is android, and as xcode is not available in the docker I got an error.

Reply:
Ok, I will move the task to you since you did it in any case